sample-contract/src/lib.rs

```diff
@@ -0,0 +1 @@
+#![no_std]\nuse soroban_sdk::{contract, contractimpl, symbol_short, vec, Env, Symbol, Vec};\n\n#[contract]\npub struct HelloContract;\n\n#[contractimpl]\nimpl HelloContract {\npub fn hello(env: Env, to: Symbol) -> Vec<Symbol> {\nvec![&env, symbol_short!(\"Hello\"), to]\n}\n}
```

Comment:
What is this ? :)

Reply:
I think there's a problem with line ending... Let me see if I can fix this.

Reply:
Ok, I think I've messed up when something, this shouldn't be there :)

Reply:
ok ! no prob. marking as resolved.